outputs.tf

```diff
@@ -22,7 +22,7 @@ output "webhook_secret" {
 # ECS
 output "task_role_arn" {
   description = "The Atlantis ECS task role arn"
-  value       = aws_iam_role.ecs_task_execution.arn
+  value       = local.ecs_task_execution_iam_role_arn
 }
 
 output "task_role_id" {
```

Comment:
the role outputs will all need to be updated to follow the `[0].x` access pattern.